Xamarin.Forms.Platform.UAP/RadioButtonRenderer.cs

```diff
@@ -9,6 +9,10 @@ namespace Xamarin.Forms.Platform.UWP
 	public class RadioButtonRenderer : ViewRenderer<RadioButton, FormsRadioButton>
 	{
 		bool _fontApplied;
+		FormsRadioButton button;
```

Comment:
No need to add another member; `Control` will be a reference to the Button if it exists.